tokenserver/tests/test_purge_old_records.py

```diff
@@ -26,19 +26,21 @@ class TestPurgeOldRecordsScript(unittest.TestCase):
     so we can test that data-deletion requests go through ok.
     """
 
-    def setUp(self):
-        super(TestPurgeOldRecordsScript, self).setUp()
-
+    @classmethod
+    def setUpClass(cls):
         # Run a testing service in a separate thread
         # so that we can test for proper calls being made.
-        self.service_requests = []
-        self.service_node = "http://localhost:8002"
-        self.service = make_server("localhost", 8002, self._service_app)
-        target = self.service.serve_forever
-        self.service_thread = threading.Thread(target=target)
-        self.service_thread.start()
+        cls.service_requests = []
+        cls.service_node = "http://localhost:8002"
+        cls.service = make_server("localhost", 8002, cls._service_app)
+        target = cls.service.serve_forever
+        cls.service_thread = threading.Thread(target=target)
+        cls.service_thread.start()
```

Comment:
This test suite spins up a localhost server on a background thread. Previously it was only used for a single test, and adding a second test caused it to start blocking and timing out, for reasons I don't really understand (maybe something not done quite right in tearing down the server?).  Anyway, rather than pull my hair out trying to track down the cause, I opted to re-use the instance across test runs, because it's trivial to clear its internal state.